### ISSUE TITLE
fixture.rkt should display stdout/err for failing tests even in non-verbose mode

### ISSUE THREAD

**Issue by glebm** [vishesh/racketscript#79]

**Comment by vishesh** [comment]
What if we dump output in separate log files? The thing I like about not producing output when running 100 tests is that we can quickly get an overview of what is happening, and then can individually look at failed tests. Since all tests are compiled, we can just use babel or traceur to run that specific JS file retained in `js-build/modules/` directory.

Another reason is that the it just produces output, but doesn't show actual test check code without which output is pretty much means nothing on that log a bunch of true/false ... outputs. The alternative would be make it somewhat more like `rackunit` style output which shows the test which failed.

**Comment by glebm** [comment]
That works as well. Or maybe a separate option like `--show-errors`? Then we can enable it on Travis to know what's going on when builds fail.

**Comment by vishesh** [comment]
Yes. `--show-errors` sounds good. On a trivial side note, we are not really showing errors (unless its crash). Mostly its Racket vs JS output, so the argument name should perhaps be different.

**Comment by vishesh** [comment]
Another way is to write a check macro which can grab the syntax location (In Rackunit style). This is particularly useful for testing RacketScript only programs.
